searx/engines/redis_server.py

```diff
@@ -0,0 +1,75 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# lint: pylint
+# pylint: disable=missing-function-docstring
+"""Redis engine (offline)
+
+"""
+
+import redis  # pylint: disable=import-error
+
+engine_type = 'offline'
+
+# redis connection variables
+host = '127.0.0.1'
+port = 6379
+password = ''
+db = 0
+
+# engine specific variables
+paging = False
+result_template = 'key-value.html'
+exact_match_only = True
+
+redis_client = None
```

Comment:
Should be `_redis_client` to avoid an error when the engine is loaded.

Reply:
Oops .. thanks! .. fixed in 39c18274